avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -626,6 +626,11 @@ Describe 'Module tests' -Tag 'Module' {
         $udtSpecificTestCases = [System.Collections.ArrayList] @() # Specific UDT test cases for singular UDTs (e.g. tags)
         foreach ($moduleFolderPath in $moduleFolderPaths) {
 
+          if ($moduleFolderPath -match '[\/|\\]avm[\/|\\]ptn[\/|\\]') {
```

Comment:
I take you don't like how the other test was skipped for ptn ([row 200](https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1#L200))? 💔

Reply:
The reason I chose this approach was only to not have to specify the exclusion for each individual test, but exclude all pattern modules from all UDT schema tests to begin with. Otherwise I'd have definitely filtered it the way you linked it

Reply:
I'll approve solely for the fact that we should restructure all tests and specs😝